README.md

```diff
@@ -84,3 +84,6 @@ function add(num1, num2) {
 * [x] Task 1
 * [x] Task 2
 * [ ] Task 3
+
+This is a line I've added :)
+
```

Comment:
@evangera This is a simple comment on Pul Request. I've added these 3 lines only for testing purposes.
Now if you press "Merge" button - all changes I've done (added 3 lines) will appear in your "master" branch.